omero/sysadmins/unix/server-centos6.txt

```diff
@@ -97,27 +81,175 @@ Install Java::
 
     yum install java-1.8.0-openjdk
 
-Install and start PostgreSQL::
+Install PostgreSQL, reconfigure to allow TCP connections and start it::
 
     yum install http://yum.postgresql.org/9.4/redhat/rhel-6-x86_64/pgdg-redhat94-9.4-1.noarch.rpm 
     yum install postgresql94-server postgresql94-contrib
     service postgresql-9.4 initdb
+    sed -i.bak -re 's/^(host.*)ident/\1md5/' /var/lib/pgsql/9.4/data/pg_hba.conf
     chkconfig postgresql-9.4 on
     service postgresql-9.4 start
 
-Running OMERO
--------------
+For the steps described above, download: 
+
+	[link to be added]
+
+Setting up the OMERO.server
+---------------------------
+This is an example walkthrough for installing OMERO on Centos6 with Python 2.7 using
+a dedicated system user.
+You can use this as a guide for setting up your own test server. For production use you should also read the pages listed under :ref:`index-optimizing-server`.
+
+For convenience in this walkthrough the main OMERO configuration options have
+been defined as environment variables. When following this walkthrough you can
+either use your own values, or alternatively source the following file :download:`settings.env <walkthrough/settings.env>`:
+
+.. literalinclude:: walkthrough/settings.env
+   :start-after: Substitute
+
+Once Python2.7, Ice3.5 and the additional packages are installed:
+
+Create an omero system user, and a directory for the OMERO repository:
+
+.. literalinclude:: walkthrough/step02_all_setup.sh
+
+| :download:`step02_all_setup.sh <walkthrough/step02_all_setup.sh>`
+
+
+Create a database user and initialize a new database for OMERO:
+
+.. literalinclude:: walkthrough/step03_all_postgres.sh
+
+| :download:`step03_all_postgres.sh <walkthrough/step03_all_postgres.sh>`
+
+To install the remaining dependencies, create a virtual environment::
+
+	virtualenv -p /usr/bin/python2.7 /home/omero/omeroenv
+	source /home/omero/omeroenv/bin/activate
+	#upgrade pip
+	/home/omero/omeroenv/bin/pip install --upgrade pip
+
 
 The following settings will need adding to your OMERO startup script
-or to the OMERO user's environment (for example in a shell startup
-script)::
+or to the omero user's environment (for example in a shell startup
+script). Add the absolute path to the :file:`bin` directory of 
+the virtual environment :file:`/home/omero/omeroenv` to the ``PATH``
+variable::
 
-    source /opt/rh/python27/enable
     ICE_HOME=/opt/Ice-3.5.1
-    PATH="${ICE_HOME}/bin:$PATH"
+    PATH="${ICE_HOME}/bin:/home/omero/omeroenv/bin:$PATH"
     export LD_LIBRARY_PATH="${ICE_HOME}/lib64:${ICE_HOME}/lib:$LD_LIBRARY_PATH"
```

Comment:
With making Ice global in https://github.com/openmicroscopy/ome-documentation/pull/1348/files#diff-7f9ce30296840990d03ea2853e956e74R73 you do not need LD_LIBRARY_PATH nor Slice